cmd/dep/init.go

```diff
@@ -43,7 +43,7 @@ func (cmd *initCommand) Register(fs *flag.FlagSet) {}
 type initCommand struct{}
 
 func trimPathPrefix(p1, p2 string) string {
-	if filepath.HasPrefix(p1, p2) {
+	if strings.HasPrefix(p1, p2) {
 		return p1[len(p2):]
 	}
 	return p1
```

Comment:
Yeah, this has been raised, and we have an extensive issue discussing it - #296. Let's leave this one as-is for now, and deal with it over there.